tests/integration_tests.rs

```diff
@@ -131,174 +133,46 @@ fn test_prove_quadratic() {
 }
 
 #[ignore = "metal"]
-#[test_log::test]
-fn test_prove_cairo_simple_program() {
-    /*
-    Cairo program used in the test:
-
-    ```
-    func main() {
-        let x = 1;
-        let y = 2;
-        assert x + y = 3;
-        return ();
-    }
+/// Loads the program in path, runs it with the Cairo VM, and amkes a proof of it
+fn test_prove_cairo_program(file_path: &str) {
+    let (register_states, memory, program_size) =
+        run_program(None, CairoLayout::Plain, file_path).unwrap();
 
-    ```
-    */
-    let (raw_trace, memory) = load_cairo_trace_and_memory("simple_program");
     let proof_options = ProofOptions {
         blowup_factor: 4,
-        fri_number_of_queries: 1,
+        fri_number_of_queries: 3,
         coset_offset: 3,
     };
 
-    let program_size = 5;
-    let mut program = vec![];
-    for i in 1..=program_size as u64 {
-        program.push(memory.get(&i).unwrap().clone());
-    }
+    // This should be auto calculated
+    let padded_trace_length = next_power_of_two(memory.len() as u64) as usize;
```

Comment:
The stdlib already gives us a method to find the [next power of 2](https://doc.rust-lang.org/std/primitive.usize.html#method.next_power_of_two);
```suggestion
    let padded_trace_length = memory.len().next_power_of_two();
```